crates/core_arch/src/aarch64/neon/mod.rs

```diff
@@ -2,6 +2,9 @@
 
 #![allow(non_camel_case_types)]
 
+mod generated;
```

Comment:
Shouldn't be necessary.

Reply:
See https://github.com/rust-lang/stdarch/pull/802 and previous discussion.